modal_docs/gen_reference_docs.py

```diff
@@ -76,20 +76,14 @@ def top_level_filter(module, name):
 
     base_title_level = "#"
     forced_module_docs = [
-        ("modal.Function", "modal.functions"),
-        ("modal.Secret", "modal.secret"),
-        ("modal.Dict", "modal.dict"),
-        ("modal.Queue", "modal.queue"),
         ("modal.call_graph", "modal.call_graph"),
         ("modal.gpu", "modal.gpu"),
         ("modal.runner", "modal.runner"),
-        ("modal.Sandbox", "modal.sandbox"),
-        ("modal.ContainerProcess", "modal.container_process"),
         ("modal.io_streams", "modal.io_streams"),
         ("modal.FileIO", "modal.file_io"),
     ]
     # These aren't defined in `modal`, but should still be documented as top-level entries.
-    forced_members = {"web_endpoint", "asgi_app", "method", "wsgi_app", "forward"}
+    forced_members: set[str] = set()
```

Comment:
Is this change intentional? Just checking that `web_endopint`, `asgi_app` and so on should still be documented somewhere.

Reply:
Yeah, I'm not sure about the context here, but these names are all in the top-level modal namespace now and get documented on my branch: 
<img width="399" alt="image" src="https://github.com/user-attachments/assets/54d4475b-a843-4f9f-8be4-04e7d9db2e94" />